js/src/interface.js

```diff
@@ -225,7 +226,8 @@ let app = new Vue({
     selectedDataSource() {
       var dataSource = this.selectedDataSource;
       this.onSelectChange();
-      if (dataSource) {
+      if (dataSource && !firstLoad) {
+        firstLoad = false;
```

Comment:
@seromenho `firstLoad` needs to be `false` to match the `if` condition. Then you're setting it as `false` again?